### PR TITLE
add recap

### PR DESCRIPTION
**Pull request by jumoog**

### CODE REVIEW

**Comment by jumoog** [comment]
just a dirty hack for now.. Analyzer is and skip part still missing

File: ConfusedPolarBear.Plugin.IntroSkipper/Controllers/SkipIntroController.cs [hunk 1]
```diff
@@ -80,6 +81,12 @@ public ActionResult UpdateTimestamps([FromRoute] Guid id, [FromBody] TimeStamps
             Plugin.Instance!.Credits[id] = new Intro(id, cr);
         }
 
+        if (timestamps?.Recap.IntroEnd > 0.0)
+        {
+            var cr = new TimeRange(timestamps.Recap.IntroStart, timestamps.Recap.IntroEnd);
+            Plugin.Instance!.Credits[id] = new Intro(id, cr);
+        }
+
```
Comment (Gauvino) [inline ConfusedPolarBear.Plugin.IntroSkipper/Controllers/SkipIntroController.cs:89]: Shouldn't this be like before intro and credits, like function for recap intro and credit to be in an order?

Comment (jumoog) [reply ConfusedPolarBear.Plugin.IntroSkipper/Controllers/SkipIntroController.cs:89]: At this point, I just store the data. The client decides what to do with the information when it is played.

**Comment by jumoog** [comment]
start working on that once 10.10 is out and we use the new segment api

**Pull request closed without merging**